src/main/resources/i18n/messages_de.properties

```diff
@@ -95,7 +93,7 @@ email.notification.aux.difficulty.hard=Schwer
 
 # Plagiarism
 email.plagiarism.title=Neuer Plagiatsfall: Übung "{0}" im Kurs "{1}"
-email.plagiarism.cpc.title=Neue signifikante �bereinstimmung: Aufgabe "{0}" im Kurs "{1}"
+email.plagiarism.cpc.title=Neue signifikante �bereinstimmung: Aufgabe "{0}" im Kurs "{1}"
```

Comment:
`Ü` is replaced by some other codepoint

Reply:
I corrected the encoding with commit 33b924c6dafd93d83197f542b74a7c6f37adfa54 for this file